CHANGELOG.md

```diff
@@ -17,11 +17,12 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 
  * Data can be imported into Datasets V2.
  * Repositories can be upgraded from V1 to V2 with `sno upgrade 02-05`.
- * A working copy can be checked out and features can be modified, such that `sno diff` shows the diff.
+ * A working copy can be checked out and features can be modified, diffed and committed.
 
 #### Important missing functionality in Datasets V2
 
- * Changes to V2 datasets cannot be committed, whether feature changes or schema changes.
+ * Meta/schema changes to V2 datasets cannot be committed or diffed.
```

Comment:
there's no way to actually make a change to a meta item at present, so I'd just leave this line out